whiteboard/whiteboard-ps/whiteboard/Get-WhiteboardOwners.md

```diff
@@ -0,0 +1,93 @@
+---
+external help file: Microsoft.Whiteboard.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftWhiteboard
+online version: https://docs.microsoft.com/powershell/module/whiteboard/get-whiteboardowners
+applicable: Microsoft Whiteboard
+title: Get-WhiteboardOwners
+schema: 2.0.0
+author: shlevari
+ms.author: shlevari
+ms.reviewer:
+---
+
+# Get-WhiteboardOwners
+
+## SYNOPSIS
+
+Gets all the users in a tenant who own whiteboards in a specified geography.
+
+## SYNTAX
+
+### Get the whiteboard owners in a geography
+
+```powershell
+Get-WhiteboardOwners -Geography <geography> -ContinuationToken <optionalToken> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Gets all the users in a tenant who own whiteboards in a specified geography. Returns them as an object containing a list of user object, a tenantId and a continuation token. By calling repeatedly passing in the new continuation tokens, all the owners for a tenant can be gathered. The data returned is precalculated and therefore not realtime. Results are precalculated approximately every two weeks.
+
+## EXAMPLES
+
+### -------------------------- EXAMPLE 1 --------------------------
+
+```
+PS C:\>Get-WhiteboardOwners -Geography Europe
+```
+
+Get the owners in the European geography.
+
+### Output
+
+```yaml
+TenantId:                  tenantId
+Geography:                 The geography queried for
+Items:                     List of user ids
+ContinuationToken:         continuation token
+```
+
+
+## PARAMETERS
+
+### -Geography
+Required. The geography to look for board owners in. Accepted values are: Europe, Australia, or Worldwide (all boards not in australia or europe).
+
+```yaml
```

Comment:
Please see this, it is the output given by PlatyPS for the Geography parameter

![image](https://user-images.githubusercontent.com/36666927/119495525-b9933400-bd1f-11eb-92ee-cf1034681eaf.png)